soroban-env-common/src/raw_val.rs

```diff
@@ -518,6 +522,18 @@ impl From<&ScError> for RawVal {
     }
 }
 
+/* impl From<ScInitialLifetime> for RawVal {
```

Comment:
Is this supposed to be here / be commented out?

Reply:
Removed